src/Api/BoardApi.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
```

Comment:
The file name should be `boredApi.js`, Capital letter refers to a React component and those API calls should just be pure functions